students/sangwoong/westagram/users/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from users.view import UserView
+from users.view import UserView, SigninView
 
 urlpatterns = [ 
-	path ('', UserView.as_view())
+	path ('users', UserView.as_view()),
```

Comment:
최종적인 엔드포인트는 users/users가 되겠네요. 어떤 기능을 하는 api와 연결되어있는지 명확하지 않고 중복적입니다. 회원가입은 일반적으로 아래와 같은 엔드포인트를 사용합니다.
```suggestion
	path ('signup', UserView.as_view()),
```